### PR TITLE
Layout Changes (v0.5)

### PR DESCRIPTION
**Pull request by JZSandbox**
Overall Design Changes, still not 100% done

### CODE REVIEW

**Review by erpofficial — APPROVED** [review]

**Comment by Branqueador** [comment]
a print of how is going or whats looking ?

**Comment by KingRyuShin** [comment]
yeah you got a picture of how it will look with your change ?

File: html_app/components/ItemSlot.vue [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-    <div :class="prefixClass + 'item-slot'" @mouseenter="$bus.trigger('setInfo', item)" @mouseleave="$bus.trigger('resetInfo')">
+    <div :class="prefixClass + 'item-slot'" @mouseenter="$bus.trigger('setInfo', item)" @mouseleave="$bus.trigger('resetInfo')"> 
         <div :class="prefixClass + 'item-slot-key'" v-if="(inventory == 'player' || inventory == 'hotbar') && (slot < 6 || slot == 41)">
             <p>{{ slot % 7 }}</p>
         </div>
         <div :class="prefixClass + 'item-slot-img'">
             <img :src="item.image" :alt="item.name" v-if="item">
         </div>
-        <div class="item-slot-label" v-if="!item || !item.isWeapon" :title="(item ? item.label : '')">
+        <div :class="'item-slot-label ' + (item ? 'isItem ' : '') + (item.isWeapon ? ' isItem ' : '')" v-if="!item || !item.isWeapon" :title="(item ? item.label : '')">
```
Comment (restray) [inline html_app/components/ItemSlot.vue:9]: A better syntax for classes would be:
```vue
:class="['item-slot-label', {'isItem': item}, {'isItem': item.isWeapon}]"
```

`item.isWeapon` will make the application crash! So think to replace it white `item && item.isWeapon`

File: html_app/components/PlayerInventory.vue [hunk 2]
```diff
@@ -3,19 +3,61 @@
         <div id="qbcore-inventory">
             <div class="inventory-info">
                 <div class="player-inv-info">
-                    <span id="player-inv-label">{{ i18n.playerInventory.player_inventory }}</span><br>
+                    <div class="player-inv-info-box">
+                        <div class="player-inv-info-box-left">
+                            <span class="material-icons">
+                            account_circle
```
Comment (restray) [inline html_app/components/PlayerInventory.vue:9]: Add an indent over here

File: html_app/components/PlayerInventory.vue [hunk 3]
```diff
@@ -3,19 +3,61 @@
...
+                            <span class="material-icons">
+                                scale
+                            </span>
+                        </div>
+                        <div class="player-inv-info-weight-right">
+                            <p>{{ (totalWeight / 1000).toFixed(2) }}</p>
+                            <p>/{{ (playerInventory.maxweight / 1000).toFixed(2)}}</p>
+                        </div>
+                    </div>
+                   <!-- <span id="player-inv-label">{{ playerInventory.playerName }}</span><br>
```
Comment (restray) [inline html_app/components/PlayerInventory.vue:27]: You can delete unused code ! Git remembers everything :D

File: html_app/components/PlayerInventory.vue [hunk 4]
```diff
@@ -3,19 +3,61 @@
...
                         ⚖️: {{ (totalWeight / 1000).toFixed(2) }} / {{ (playerInventory.maxweight / 1000).toFixed(2) }}
-                    </span>
+                    </span>-->
                 </div>
                 <div class="other-inv-info">
-                    <span id="other-inv-label">{{ openedInventory.label }}</span><br>
+                    <div class="other-inv-info-box">
+                        <div class="other-inv-info-box-left">
+                            <span class="material-icons">
+                            account_circle
```
Comment (restray) [inline html_app/components/PlayerInventory.vue:42]: Another tab over here :)

File: html_app/components/PlayerInventory.vue [hunk 5]
```diff
@@ -3,19 +3,61 @@
...
+                            <span class="material-icons">
+                                scale
+                            </span>
+                        </div>
+                        <div class="other-inv-info-weight-right">
+                            <p>{{ (totalWeightOther / 1000).toFixed(2) }}</p>
+                            <p>/{{ (playerInventory.maxweight / 1000).toFixed(2)}}</p>
+                        </div>
+                    </div>
+                    <!--<span id="other-inv-label">{{ openedInventory.label }}</span><br>
```
Comment (restray) [inline html_app/components/PlayerInventory.vue:60]: Another commented code there !

File: html_app/components/PlayerInventory.vue [hunk 6]
```diff
@@ -67,10 +109,13 @@
             </div>
             <item-info />
             <div class="inv-background"></div>
+
```
Comment (restray) [inline html_app/components/PlayerInventory.vue:112]: Can we drop those blank lines?

File: html_app/components/PlayerInventory.vue [hunk 7]
```diff
@@ -67,10 +109,13 @@
             </div>
             <item-info />
             <div class="inv-background"></div>
+            
         </div>
     </div>
 </template>
 
+
+
```
Comment (restray) [inline html_app/components/PlayerInventory.vue:118]: Those too?

File: html_app/index.css [hunk 8]
```diff
@@ -486,20 +474,18 @@ body {
...
-	flex: 0 0 5vw;
+    display: block;
+    position: relative;
+    margin-right: 0.25vw;
+    margin-left: 0.25vw;
+    width: 12vh;
+    transform: translateX(0);
+    height: 12vh;
+    opacity: 1;
+    /* flex: 0 0 5vw; */
```
Comment (restray) [inline html_app/index.css:485]: You can delete it too!

File: html_app/index.css [hunk 9]
```diff
@@ -959,3 +948,328 @@ body {
 .weapon-attachments-back:hover {
 	background-color: rgb(18, 18, 18);
 }
+
+/*
+Colors
+*/
+/* Item Slots */
+#qbcore-inventory {
```
Comment (restray) [inline html_app/index.css:956]: Please indent all the lines below correctly (magic tips: VSCode CTRL+Shift+P -> Format file)

File: html_app/index.css [hunk 10]
```diff
@@ -959,3 +948,328 @@ body {
...
+	background-color: crimson;
+	font-family: 'Poppins', sans-serif;
+	border: 1px solid #DC143C;
+	border-radius: 5px;
+	-webkit-box-shadow: 0 3px 6px rgba(0, 0, 0, 0.39);
+			box-shadow: 0 3px 6px rgba(0, 0, 0, 0.39);
+  }
+  
+  .player-inv-info-box {
+	display: -webkit-box;
```
Comment (restray) [inline html_app/index.css:1032]: Specifiying webkit and mozilla support is useless for use, Chromium is the only engine that could run any CSS! So we can delete it (just a suggestion)

**Comment by restray** [comment]
image
Having this weird bug too... Flexbox isn't configure well or tiles are setup with fixed values!

File: html_app/index.html [hunk 11]
```diff
@@ -2,9 +2,9 @@
 <html>
 <head>
   <meta charset="utf-8">
-  <title></title>
-  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
-  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <title>QB-Inventory</title>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet"> 
+  <link href="https://fonts.googleapis.com/css?family=Material+Icons|Material+Icons+Outlined|Material+Icons+Two+Tone|Material+Icons+Round|Material+Icons+Sharp" rel="stylesheet">
```
Comment (restray) [inline html_app/index.html:7]: We can have one link (so one request) for both of them !
`https://fonts.googleapis.com/css?family=Material+Icons|Material+Icons+Outlined|Material+Icons+Two+Tone|Material+Icons+Round|Material+Icons+Sharp|Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap`

**Review by restray — CHANGES_REQUESTED** [review]
The update is almost complete, need to patch a few things then it's ready to go :D Thanks for your beautiful job :)

File: html_app/index.css (hunk 9 above)
Comment (JZSandbox) [reply html_app/index.css:956]: Its coming from the scss, that too doesnt matter

**Comment by SonofGlensow** [comment]
Can you add money to inventory?

**Pull request merged**